pyccel/codegen/python_wrapper.py

```diff
@@ -127,6 +127,8 @@ def create_shared_library(codegen,
 
         setup_filename = os.path.join(pyccel_dirpath, setup_filename)
         cmd = [sys.executable, setup_filename, "build"]
+        if sys.platform == 'win32' and compiler in ('gfortran','gcc','ifort','icc'):
+            cmd+=['--compiler=mingw32']
```

Comment:
A bit of whitespace around `+=` is not forbidden :wink:

Reply:
Fixed in 8bd329d